NServiceBus.Serilog.Tracing/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'll remove both app.configs, they were' added automatically by nuget when the packages were updated.  Probably early am tomorrow when I get in the office.